lua/astrocommunity/split-and-window/mini-map/init.lua

```diff
@@ -0,0 +1,39 @@
+return {
+  {
+    "echasnovski/mini.map",
+    dependencies = { "lewis6991/gitsigns.nvim", optional = true },
+    version = "*",
```

Comment:
this is unnecessary

Reply:
this is actually a requirement from @mehalter going forward https://github.com/AstroNvim/astrocommunity/pull/664 -- I guess it should be in the docs going forward to avoid confusion?

Reply:
This optional thing is implemented incorrectly. It should only add the gitsigns integration if it's available. I can help with this later today and fix this pack